layers/+os/osx/keybindings.el

```diff
@@ -58,8 +61,9 @@ other than the three sane values listed above."
                         '((hyper . "H-")
                           (super . "s-")
                           (alt   . "A-")))))
-      (when found (kbd (concat (cdr found) keys)))))
-
+      (if found
+        (kbd (concat (cdr found) keys))
```

Comment:
Could you fix the indentation here, by default the `then` statement is indented more than the `else` statement.
```elisp
      (if found
          (kbd (concat (cdr found) keys))
        (kbd (concat "H-" keys)))))
```

And add a newline between the if statement and the following comment:
>  ;; Keybindings